internal/integration/integration_test.go

```diff
@@ -0,0 +1,66 @@
+// Package integration contains genqlient's integration tests, which run
+// against a real server.
+//
+// These are especially important for cases where we generate nontrivial logic,
+// such as JSON-unmarshaling.
```

Comment:
You should add a comment pointing about where the test server code is.  Otherwise the magic test data appears out of nowhere.